components/twitter/actions/like-tweet/like-tweet.mjs

```diff
@@ -0,0 +1,37 @@
+import twitter from "../../twitter.app.mjs";
+
+export default {
+  key: "twitter-like-tweet",
+  name: "Like Tweet",
+  description: "Like the tweet specified specified in the ID parameter",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    twitter,
+    tweetID: {
+      propDefinition: [
+        twitter,
+        "tweetID",
+      ],
+    },
+    includeEntities: {
+      propDefinition: [
+        twitter,
+        "includeEntities",
+      ],
+    },
+  },
+  async run() {
+    const {
+      tweetID,
+      includeEntities,
+    } = this;
+
+    const params = {
+      id: tweetID,
+      includeEntities,
+    };
+
+    return await this.twitter.likeTweet(params);
```

Comment:
Since this is an async function. [You don't need to `return await`](https://eslint.org/docs/rules/no-return-await). Could you please remove it?